src/sentry/integrations/aws_lambda/utils.py

```diff
@@ -313,6 +314,20 @@ def get_invalid_layer_name(err_message):
     return None
 
 
+def get_too_many_requests_error_message(err_message):
+    """
+    Check to see if an error matches the Too many requests error text
+    :param err_message: error string
+    :return boolean value if the error matches the too many requests error
+    """
+    too_many_requests_err = (
+        "An error occurred (TooManyRequestsException) when calling the "
+        "UpdateFunctionConfiguration operation (reached max retries: 4): "
```

Comment:
@ahmedetefy I think we should make this check a bit more agnostic to the exact text. IMO, if we see a `TooManyRequestsException` in the string, that's enough to determine we hit this case.